make_profiler/report_export.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+
+from datetime import datetime
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+
+def escape_json_string(s):
+    s=str(s)
```

Comment:
Just a formatting note. In some places spaces around operators missing. If you use Pycharm you can fix this easily by selecting all the text and using hot key `ctrl+alt+L` (`cmd+option+L` for macOs). This will fix all the formatting issues.